Signal/src/ViewControllers/MessageDetailViewController.swift

```diff
@@ -83,6 +87,22 @@ class MessageDetailViewController: OWSViewController, MediaGalleryDataSourceDele
             object: OWSPrimaryStorage.shared().dbNotificationObject)
     }
 
+    override public func viewWillLayoutSubviews() {
+        Logger.debug("\(self.logTag) in \(#function)")
+
+        super.viewWillLayoutSubviews()
+
+        self.conversationLayoutInfo.viewWidth = self.view.width()
```

Comment:
Most of the time the view size isn't changing. Would `viewWillTransition:toSize:` be a better candidate?

Reply:
Sure, that should be equivalent.